src/handlers.rs

```diff
@@ -68,83 +68,118 @@ pub fn pre_prepare(
 ) -> Result<(), PbftError> {
```

Comment:
It looks like `handlers.rs` is only used by `node.rs`. Given that we're refactoring out bits of `node.rs` into smaller functions over in that file, should we just uproot and move `handlers.rs` entirely, or move all of the refactored smaller functions over here?